services/pdf-ms/templateExample/ekb-löpande.json

```diff
@@ -0,0 +1,480 @@
+{
+  "numberOfpages": 5,
+  "defaultFontSize": 11,
+  "texts":
+  [
+  {
+      "page": 0,
+      "x": 400,
+      "y": 685,
+      "size": 11,
+      "text": "{{valuesChanged}}",
+      "valueId": "valuesChanged"
+  },
+  {
+      "page": 0,
+      "x": 270,
+      "y": 648,
```

Comment:
maybe left, top

Reply:
I think x,y is pretty clear, and probably more standard than anything else for giving coordinates? And it's what the pdf library uses.

Reply:
> I think x,y is pretty clear, and probably more standard than anything else for giving coordinates? And it's what the pdf library uses.

standard for who?! An editor | ux-person | developer | me | you